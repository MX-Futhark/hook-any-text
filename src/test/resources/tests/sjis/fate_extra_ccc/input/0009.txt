94FC93F7814182C682A282A491A28CEA82AA945D93E082C994AD90B682B782E98142000000000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D0000000E0000000F000000100000040000002C15C90800007041000084420000803F0000803F0000000000002000270000002800000080000000FFFFFFFF0002000000010000FFFFFFFF0F000000180000009014C9081814C908C005A34800000000000000009056914E82C893F782F08CA982C493F790488F6282AA235255425382D382E9235255424595B12352454E4482A297A782C282CC82CD81410A82BB82EA82AA8CDC8AB482B782D782C482C98167235255425382A482DC23525542458E7C2352454E4482A2816882C6936082A682E982A982E782BE814200000000000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D0000000E0000000F000000100000001100000012000000130000001400000015000000160000001700000018000000190000001A0000001B0000001C0000001D0000001E0000001F0000002000000021000000220000002300000024000000250000002600000B0000000000000000000000000000000000000000001F01000000000000000000000000FFFFFFFF0080A7430000A642000000000000604100000000FFFFFFFF0080AE430000A642000000000000000000006041FFFFFFFF0080A7430000C242000000000000604100006041FFFFFFFF0080AE430000C24200000000A015A34802000000CC15C908000000000000000000000000FFFFFFFF000000006814A34802000000EC15C908000000000000000000000000FFFFFFFF00010000D014A348020000000C16C908000000000000000000000000FFFFFFFF000100003815A348020000002C16C908000000000000000000000000FFFFFFFF00010000A015A3480200000000000000000000000000000000000000FFFFFFFF00010000