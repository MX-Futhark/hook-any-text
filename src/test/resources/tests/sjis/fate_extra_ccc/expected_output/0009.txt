["美肉、という造語が脳内に発生する。",
"新鮮な肉を見て肉食獣が奮い立つのは、
それが五感すべてに“旨い”と伝えるからだ。"]